src/agent/onefuzz-telemetry/src/lib.rs

```diff
@@ -161,14 +215,53 @@ impl EventData {
             Self::ToolName(x) => ("tool_name", x.to_owned()),
             Self::Region(x) => ("region", x.to_owned()),
             Self::Role(x) => ("role", x.as_str().to_owned()),
+            #[cfg(feature = "intel_instructions")]
+            Self::MissedInstructionCode(x) => ("missed_instruction_code", format!("{:?}", x)),
```

Comment:
Double-checking: we really want `:?`, here and below?

Reply:
Yes.  iced_x86::Code and iced_x86::Mnemonic do not implement Display.  However, they do implement fmt::Debug which provides what we want:

see iced_x86::Code:
https://github.com/icedland/iced/blob/315750d2ed73638c9bd5a687df6b5eeedd2fb1d1/src/rust/iced-x86/src/code.rs#L38859-L38864

See iced_x86::Mnemonic:
https://github.com/icedland/iced/blob/315750d2ed73638c9bd5a687df6b5eeedd2fb1d1/src/rust/iced-x86/src/mnemonic.rs#L3280-L3285